proto/logger/logger_connect.d.ts

```diff
@@ -0,0 +1,26 @@
+// @generated by protoc-gen-connect-es v1.4.0 with parameter "keep_empty_files=true,target=js+dts"
```

Comment:
do these need to be checked in with rules_js?

Reply:
There is a mode to generate them on the fly that wouldn't work out of the box for me. I'll give it another quick try.

Reply:
Got it working.